### DISCUSSION TITLE
configureChains set default chain polygonMumbai ?

### DISCUSSION THREAD

**Discussion by champseo2017** [wevm/wagmi#1687] [Q&A]
When initializing connect matamask I would like to set chain default to polygonMumbai . 
code: 
const chains = [
  arbitrum,
  avalanche,
  bsc,
  fantom,
  mainnet,
  optimism,
  polygon,
  polygonMumbai
];

**Comment by SaadNiaziDev** [comment]
<Web3Modal
          projectId={walletConnectProjectId}
          ethereumClient={ethereumClient}
defaultChain: polygonMumbai;
/>
